### PR TITLE
Google places integration w/ Search View

### PR DESCRIPTION
**Pull request by mbatilando**
Issue #43

### CODE REVIEW

File: Mapstagram/GooglePlaces.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+//  GooglePlaces.swift
+//  Mapstagram
+//
+//  Created by Mari Batilando on 3/12/15.
+//  Copyright (c) 2015 JMK. All rights reserved.
+//
+
+import Foundation
+
+class GooglePlacesClient {
```
Comment (jamietsao) [inline Mapstagram/GooglePlaces.swift]: Can you move this into the API directory for consistency?  That way the main directory is clean.

File: Mapstagram/Location.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+//  Location.swift
+//  Mapstagram
+//
+//  Created by Mari Batilando on 3/13/15.
+//  Copyright (c) 2015 JMK. All rights reserved.
+//
+
+import Foundation
+
+class Location {
```
Comment (jamietsao) [inline Mapstagram/Location.swift]: Same comment.  Move to Models directory.

Comment (mbatilando) [inline Mapstagram/Location.swift]: screen shot 2015-03-13 at 3 23 57 pm

Wut.

Comment (jamietsao) [inline Mapstagram/Location.swift]: That's just the view in XCode.  You can group things in XCode but unless you explicitly create directories on the file system, it'll put them all in the top level directory.  That's why Kristen moved some files yesterday and warned of potential merge conflicts.

Comment (mbatilando) [inline Mapstagram/Location.swift]: Ah, gotcha. Let me finish up integrating the search view with the location singleton and I'll put it in my next PR. I wanted to go ahead work on that so I merged this so I wouldn't have conflicts

Comment (jamietsao) [inline Mapstagram/Location.swift]: :+1:

**Pull request merged**